RecoMuon/MuonSeedGenerator/plugins/MuonSeedGenerator.cc

```diff
@@ -107,8 +107,18 @@ void MuonSeedGenerator::produce(edm::Event& event, const edm::EventSetup& eSetup
 void MuonSeedGenerator::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
   desc.setAllowAnything();
+  desc.add<edm::InputTag>("beamSpotTag", edm::InputTag("offlineBeamSpot"));
+  desc.add<bool>("scaleDT", true);
+  desc.add<edm::InputTag>("CSCRecSegmentLabel", edm::InputTag("cscSegments"));
+  desc.add<edm::InputTag>("DTRecSegmentLabel", edm::InputTag("dt4DSegments"));
+  desc.add<edm::InputTag>("ME0RecSegmentLabel", edm::InputTag("me0Segments"));
   desc.add<bool>("EnableDTMeasurement", true);
   desc.add<bool>("EnableCSCMeasurement", true);
   desc.add<bool>("EnableME0Measurement", false);
+  desc.add<std::vector<double>>("crackEtas", {0.2, 1.6, 1.7});
+  desc.add<double>("crackWindow", 0.04);
+  desc.add<double>("deltaPhiSearchWindow", 0.25);
+  desc.add<double>("deltaEtaSearchWindow", 0.2);
+  desc.add<double>("deltaEtaCrackSearchWindow", 0.25);
   descriptions.add("produceMuons", desc);
```

Comment:
At this point, the default config could be more meaningfully renamed "muonSeedGenerator"